docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -0,0 +1,1249 @@
+        definition to indicate which hosts are associated with it. There are
+        two options:</para>
+
+        <itemizedlist>
+          <listitem>
+            <para><emphasis role="bold">hostGroup: &lt;name&gt;</emphasis> The
+            name of the host group for bare metal. The name of the hostGroup
+            must match the name of the storage plane.</para>
+          </listitem>
+
+          <listitem>
+            <para><emphasis role="bold">hosts:
+            &lt;list-of-namesname&gt;</emphasis> An inline list of hosts.
+            Primarily useful for defining one-off external landing
+            zones.</para>
+          </listitem>
+        </itemizedlist>
+
+        <para>For Example:</para>
+
+        <programlisting>storage: 
+  planes: 
+  - name: demoOne 
+    category: data 
+    prefix: "/home/demo/temp" 
+    hostGroup: groupABCD # The name of the hostGroup 
+  - name: myDropZone 
+    category: lz 
+    prefix: "/home/demo/mydropzone" 
+    hosts: [ 'mylandingzone.com' ] # Inline reference to an external host. </programlisting>
+      </sect3>
+
+      <sect3 id="YAML_Persist_storage">
+        <title>Persistent Storage</title>
+
+        <para>Kubernetes uses persistent volume claims (pvcs) to provide
+        access to data storage. HPCC Systems supports cloud storage through
+        the cloud provider that can be exposed through these persistent volume
+        claims.</para>
+
+        <para>Persistent Volume Claims can be created by overriding the
+        storage values in the delivered Helm chart. The values in the
+        <emphasis
+        role="boldIt">examples/local/values-localfile.yaml</emphasis> provided
+        override the corresponding entries in the original delivered stock
+        HPCC Systems helm chart. The localfile chart creates persistent
+        storage volumes. You can use the values-localfile.yaml directly (as
+        demonstrated in separate docs/tutorials) or you can use it as a basis
+        for creating your own override chart.</para>
+
+        <para>To define a storage plane that utilizes a PVC, you must decide
+        on where that data will reside. You create the storage directories,
+        with the appropriate names and then you can install the localfiles
+        Helm chart to create the volumes to use the local storage option, such
+        as in the following example:</para>
+
+        <programlisting>helm install mycluster hpcc/ -f examples/local/values-localfile.yaml</programlisting>
+
+        <para>Note: The settings for the PVC's must be ReadWriteMany, except
+        for Dali which can be ReadWriteOnce.</para>
+
+        <para>There are a number of resources, blogs, tutorials, even
+        developer videos that provide step-by-step detail for creating
+        persistent storage volumes.</para>
+      </sect3>
+
+      <sect3 id="YML_TheDataCategory">
+        <title>Cluster Disambiguation</title>
```

Comment:
I'm not quite sure how this relates to the rest of the document.

Reply:
does this even belong in this document?  
perhaps in the beginning section?